generate_docs.sh

```diff
@@ -7,4 +7,4 @@ sed -i 's/CONTRIBUTING.md/\/contributing/' docs/index.md
 sed -i 's/docs\/using-snapshot-version.md/using-snapshot-version/' docs/index.md
 
 # Build the docs
-./gradlew clean dokka
+./gradlew clean dokkaHtmlMultiModule
```

Comment:
You don't need multi module here, but it doesn't hurt it use it. Also you've switched from gfm to html docs here, so you might need to tweak the publish script

Reply:
Ah yes, will change to `dokkaGfmMultiModule`